### ISSUE TITLE
Select multiple auto resize from content could also gain from max-content

### ISSUE THREAD

**Issue by brunoais** [craigfrancis/iframe-height#3]
See the bugzilla bug I filled years ago:
https://bugzilla.mozilla.org/show_bug.cgi?id=649970

I also filed a bug somewhere in https://www.w3.org/Bugs/Public/ but I can't find it now. It's been too many years ago.
I can't find it migrated here either: https://github.com/whatwg/html/issues/

Currently select has the attribute `size` which defaults to 4. I think this could be interesting to override that value.
Basically, it would be the behavior of firefox back in firefox 3.5.
